docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,131 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on Minikube, the CPU cluster, and GPU cluster. We adopt
```

Comment:
Do we have a minikube-managed GPU cluster?

Reply:
Is your purpose here to mention three Kubernetes clusters - a Minikube cluster, an on-premise CPU cluster managed by a tailored Kubernetes system, and a GPU cluster managed by the same tailored Kubernetes system?

Reply:
> Do we have a minikube-managed GPU cluster?

Not yet. Now, we start minikube on the PC without  GPU.

Reply:
> Is your purpose here to mention three Kubernetes clusters - a Minikube cluster, an on-premise CPU cluster managed by a tailored Kubernetes system, and a GPU cluster managed by the same tailored Kubernetes system?

Yes. And I should use more detailed section headers.